scripts/hid_configurator/configurator_cli.py

```diff
@@ -45,7 +45,13 @@ def perform_dfu(dev, args):
         return
     img_ver_dev = info.get_fw_version()
 
-    img_file = DfuImage(args.dfu_image, info, dev.get_board_name(), dev.get_complete_module_name('dfu')[len('{}/'.format('dfu')):])
+    complete_dfu_module_name = dev.get_complete_module_name('dfu')
+
+    if complete_dfu_module_name == 'dfu':
+        complete_dfu_module_name = 'not_supported'
```

Comment:
I prefer to simply set it to None. Easier to understand and lower chance of bugs